src/store/store-config.js

```diff
@@ -0,0 +1,16 @@
+import {
+  createStore,
+  combineReducers,
+  applyMiddleware,
+  compose,
+} from 'redux';
+import thunk from 'redux-thunk';
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose; // eslint-disable-line no-undef
+
+export const store = createStore(
+  combineReducers(
+    {},
```

Comment:
It’s better to create default state variable above and pass it here instead of passing curly braces. Some people can’t know what is the first argument for